VimeoUpload/Extensions/AFURLSessionManager+Extensions.swift

```diff
@@ -42,6 +42,6 @@ extension AFURLSessionManager
     @available(iOS 8.0, *)
     func taskForIdentifier(identifier: Int) -> NSURLSessionTask?
     {
-        return self.tasks.filter{ $0.taskIdentifier == identifier }.first as? NSURLSessionTask
+        return self.tasks.filter{ $0.taskIdentifier == identifier }.first as NSURLSessionTask!
```

Comment:
We should revert this change, so that we don't crash if for some reason the cast fails...